menas/ui/package.json

```diff
@@ -9,7 +9,7 @@
     "@ui5/cli": "^2.8.1"
   },
   "config": {
-    "apiUrl": "${API_URL}",
+    "apiUrl": "http://localhost:8090/rest_api_war",
```

Comment:
Keep this change local, don't commit it please 😉

Reply:
Okay, I actually forgot about that, will fix it